CONTRIBUTING.md

```diff
@@ -121,15 +121,20 @@ The full Meilisearch release process is described in [this guide](https://github
 Depending on the developed feature, you might need to provide a prototyped version of Meilisearch to make it easier to test by the users.
 
 The prototype name must follow this convention: `prototype-X-Y` where
-- `X` is the feature name formatted in `kebab-case`
+- `X` is the feature name formatted in `kebab-case`. It should not end with a number segment.
```

Comment:
with a number segment?

Reply:
yes, a component of the `kebab-cased` name that only contains something that can be parsed as a number.

There's an example below.

Reply:
nah i don't think this has to do with not knowing the phrase, it is a bit confusing indeed.

Do you have a suggestion on how I could better express this?

Reply:
Hum... 🤔 "It should end with a single number" -> not exactly true, but with the example below, we would understand `0-0` will not work